Gemeinsamer Bibliotheksverbund ISBN.js

```diff
@@ -62,6 +62,10 @@ function doSearch(item) {
 					mimeType: "application/pdf"
 				}];
 			}
+			//delete [u.a.] from place
+			if (item.place) {
+				item.place = item.place.replace(/\[u\.a\.\]\s*$|u\.a\.\s*$/, '');
```

Comment:
is u.a. standardized or might we see u. a.? I'd guess it'd be worthwhile to add a \\s? (or even [\\s\u00A0]? after the first period.
Also, why not just put ? After the [ and ] and save the part after |